src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,51 @@
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { useCallback, useEffect, useState } from 'react';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage = () => {
+  const [error, setError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const fetchPeopleFromApi = useCallback(async () => {
+    try {
+      const todosFromApi = await getPeople();
+
+      setIsLoading(false);
+      setPeople(todosFromApi);
```

Comment:
Consider moving `setIsLoading(false);` after `setPeople(todosFromApi);` to ensure that the loading state is only set to false after the data has been successfully fetched and set. This will prevent any flicker of the loading indicator if setting the people state takes some time.